src/test/groovy/org/prebid/server/functional/tests/BidValidationSpec.groovy

```diff
@@ -212,4 +212,33 @@ class BidValidationSpec extends BaseSpec {
         def metrics = defaultPbsService.sendCollectedMetricsRequest()
         assert metrics["adapter.generic.requests.bid_validation"] == initialMetricValue + 1
     }
+
+    def "PBS should return error invalid request format request.ext.aliasgvild unknown bidder alias"() {
```

Comment:
Not clear name. Use pattern "Should ExpectedBehavior When StateUnderTest" for test naming